win32ss/user/user32/controls/ghost.c

```diff
@@ -0,0 +1,570 @@
+/*
+ * PROJECT:         ReactOS user32.dll
```

Comment:
Using the latest license header as indicated in https://reactos.org/wiki/Coding_Style#File_Structure would be good.
However at the very least you need to specify a license please.

Reply:
OK